ci-operator/config/openshift-priv/kubernetes/openshift-priv-kubernetes-release-4.18.yaml

```diff
@@ -471,6 +471,9 @@ tests:
       ZONES_COUNT: "3"
     test:
     - chain: openshift-qe-control-plane
+    - ref: openshift-qe-orion-cluster-density
```

Comment:
can we use the `orion-consolidated` chain here as well as used in some of the similar places?

Reply:
yeah, I am testing this right now to ensure, but I will update all calls to orion with this.